src/features/web-compat.js

```diff
@@ -67,7 +67,7 @@ function cleanShareData (data) {
     return dataToSend
 }
 
-export default class WebCompat extends ContentFeature {
+export class WebCompat extends ContentFeature {
     /** @type {Promise<any> | null} */
```

Comment:
because of the module augmentation, we need to expose this class separately as well as default

Reply:
What happens if a feature author does this wrong?

Reply:
I think we should follow with a unit-test for this (as you suggested) - just in the interest of this PR not getting any larger.